.github/workflows/server_compatibility.yaml

```diff
@@ -379,6 +379,16 @@ jobs:
           repository: hazelcast/hazelcast-csharp-client
           path: tag
           ref: ${{ matrix.client_tag }}
+            
```

Comment:
It makes sense and I believe it work but I'm not sure that latest script can work with older versions like 4.x, fyi.

Reply:
v4 client seems out of scope of compat suites anyway ?